docs/cvl/methods.md

```diff
@@ -748,8 +748,10 @@ supported.
 The function call may also use the special variable `calledContract`, which
 gives the address of the contract on which the summarized method was called.
 This is useful for identifying the called contract in {ref}`wildcard summaries
-<cvl2-wildcards>`.  The `calledContract` keyword is only defined in the `methods`
-block.
+<cvl2-wildcards>`, both for external and _internal_ functions. Note that for
+library functions the `calledContract` is the contract _calling_ the library
```

Comment:
And for internal functions which are not in libraries this isn't true?

Reply:
For internal non-library functions it is both the called and calling contract (since they are the same).
Should we state this?

Reply:
IMO it's worth emphasizing that the `calledContract` is the contract itself. The same is true for library functions (internal library functions are a part of the contract bytecode).